test/auth/authenticate.test.js

```diff
@@ -0,0 +1,64 @@
+import { getADGroups } from '../../app/auth/authenticate.js'
```

Comment:
jest isnt installed on this repo, i think we are using mocha.

(Personally i much prefer to use jest)